piwik.php

```diff
@@ -16,6 +16,21 @@
 // Note: if you wish to debug the Tracking API please see this documentation:
 // http://developer.piwik.org/api-reference/tracking-api#debugging-the-tracker
 
+if ($_SERVER['REQUEST_METHOD'] === 'HEAD') {
```

Comment:
`$_SERVER['REQUEST_METHOD']` may not always be assigned I think. Probably it is but just to be sure I would maybe add a `!empty($...)`